src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -421,9 +421,13 @@
 
           <!-- Compression Mode -->
           <div class="form-group row">
-            <label i18n
-                   class="cd-col-form-label"
-                   for="mode">Mode</label>
+            <label class="cd-col-form-label"
+                   for="mode">
+              <ng-container i18n>Mode</ng-container>
+              <cd-helper>
+                <span i18n>Policy used for compression algorithm</span>
+              </cd-helper>
+            </label>
```

Comment:
Are the `<ng-container>` and `<span>` tags even needed here?

Reply:
Yeah, that could be probably removed.
```suggestion
            <label class="cd-col-form-label"
                   for="mode"
                   i18n>Mode
              <cd-helper i18n>
                   Policy used for compression algorithm
              </cd-helper>
            </label>
```

Reply:
Fixed it, thanks!